src/backend/config.ts

```diff
@@ -14,7 +14,7 @@ export const AUTO_BACKUP_TIMEOUT = 1000 * 60 * 10; // 10 minutes
 export const dbConfig: IDBVersioningConfig[] = [
   {
     // Version 4, 19-9-20: Added system created date
-    version: 4,
+    version: 5,
```

Comment:
@RvanderLaan ???

Reply:
oh woops, good catch, that shouldn't be in here. Updated the version in https://github.com/allusion-app/Allusion/pull/306/files and had to change it here after switching back to this branch

```suggestion
    version: 4,
```